brake/backends/cachebe.py

```diff
@@ -71,7 +73,7 @@ def limit(self, func_name, request,
             if ':ip:' in counter:
                 ratelimited_by = 'ip'
 
-            if counters[counter] > count:
+            if counters[counter][0] > count:
```

Comment:
It's not a big deal, but we're moving from a scalar value that is referred to by its name, to an opaque repeated data structure and blindly grabbing whatever is in its first index. 

Now, after talking to you, I know what that is. But this these are called "magic numbers" for a reason, and that's because they omit context.

It's a nit of a comment, if  you really feel strongly about it that's fine. I don't feel strongly; I just thought I'd bring it up.